modules/dev-tools/ts-transform-version-inline/index.cjs

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function (program, pluginConfig, { ts }) {
```

Comment:
Ditto.

Can these go into a ts-plugins directory? Is it necessary to have them in the root?